apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -427,10 +428,11 @@ class ReviewTab extends Component {
           <div style={styles.reviewHeader}>
             {viewAs !== ViewType.Teacher &&
               !errorLoadingReviewblePeers &&
+              codeReviewEnabled &&
```

Comment:
nit: as this set of conditions is growing, could we pull this out into a local `canViewPeerProject` or something?

Reply:
I pulled these checks out into the functions that already render them. Let me know if you think that duplicates too much code